ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -419,6 +419,26 @@ public boolean getAnimateMeasurementChange() {
         return animateMeasurementChange;
     }
 
+    /**
+     * Adds a custom {@link android.animation.Animator.AnimatorListener} to listen to animator
+     * update events used by this view.
+     *
+     * @param animatorListener the custom animator listener.
+     */
+    public void addAnimatorListener(Animator.AnimatorListener animatorListener) {
+        animator.addListener(animatorListener);
```

Comment:
Just to be safe - can we make the `animator` field declaration `private final ValueAnimator = ValueAnimator.ofFloat(1f);` and get rid of the assignment in `init`?